scripts/pylib/twister/twisterlib.py

```diff
@@ -2750,6 +2760,12 @@ def run(self):
             if instance.handler.type_str == "device":
                 instance.handler.suite = self.suite
 
+            if(self.seed is not None and instance.platform.name == "native_posix"):
```

Comment:
I think there's also `native_posix_64` that's valid here. Can we just check that `instance.platform.name` begins with `native_posix`?